game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -0,0 +1,309 @@
+-- Taken from bb template
+if Duels == nil then
+  DebugPrint ( 'Creating new Duels object.' )
+  Duels = class({})
+
+  Debug.EnabledModules['duels:*'] = true
+  Debug.EnabledModules['zonecontrol:*'] = true
```

Comment:
Shouldn't we enable `zonecontrol` debugging somewhere else?
And what about having a single place where Debugging get enabled? Like in `gamemode.lua` where the modules are initialized?